requirements.txt

```diff
@@ -29,8 +29,9 @@ numpy
 scipy
 scikit-learn
 
-# Training visualisation using tensorboard.
-tensorboard==1.0.0a6
+# Write logs for training visualisation with the Tensorboard application
+# Install the Tensorboard application separately (part of tensorflow) to view them.
+tensorboardX==1.0
```

Comment:
Do we need to pin this exactly to 1.0?  I'm not sure what versions this is compatible with, but it seems like we should at least do something like `>=1.0,<1.1` (maybe even `<2.0`), to give some flexibility to downstream users.

Reply:
I'm actually a fan of pinning this - it's randomly broken a few times and was in fact the original reason why I switched away from tensorboardX in the first place.